### PR TITLE
ENGDOCS-1833

### PR DESCRIPTION
**Pull request by aevesdocker**
Builds on https://github.com/docker/docs/pull/18667 

Also adds it to the new Security section

### CODE REVIEW

File: content/desktop/extensions/private-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+- `admin-settings.json`, which activates the private marketplace feature once it’s applied to Docker Desktop on your developers’ machines
+- `extensions.txt`, which determines which extensions to list in your private marketplace.
+
+## Step two: Set the private marketplace behaviour
+
+The generated `admin-settings.json` file includes various settings you can modify.
+
+Each setting has a `value` that you can set, including a `locked` field that allows you to lock the setting and make it unchangeable by your developers.
+
+- `extensionsEnabled` enables Docker Extensions
```
Comment (Frezzle) [inline content/desktop/extensions/private-marketplace.md:72]:
```suggestion
- `extensionsEnabled` enables Docker Extensions.
```

**Review by Frezzle — COMMENTED** [review]
Some small changes, otherwise it looks great, thanks for your changes Allie.

**Comment by aevesdocker** [comment]
Thanks @Frezzle, let me know when you and the Publisher team want this merged

File: content/desktop/extensions/private-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+The generated `admin-settings.json` file includes various settings you can modify.
+
+Each setting has a `value` that you can set, including a `locked` field that allows you to lock the setting and make it unchangeable by your developers.
+
+- `extensionsEnabled` enables Docker Extensions.
+- `extensionsPrivateMarketplace` activates the private marketplace and ensures Docker Desktop connects to content defined and controlled by the administrator instead of the public Docker marketplace.
+- `onlyMarketplaceExtensions` allows or blocks users from installing other extensions by using the command line. Teams developing new extensions must have this setting unlocked (`"locked": false`) to install and test extensions being developed.
+- `extensionsPrivateMarketplaceAdminContactURL` defines a contact link for users to request new extensions in the private marketplace. If `value` is empty then no link is shown to your developers on Docker Desktop, otherwise this can be either an HTTP link or a “mailto:” link. For example,
+
+   ```json
```
Comment (Frezzle) [inline content/desktop/extensions/private-marketplace.md:77]: oops, the ending "```" got stripped somehow

File: content/desktop/extensions/private-marketplace.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
+---
+description: How to configure and use Docker Extensions' private marketplace
+keywords: Docker Extensions, Docker Desktop, Linux, Mac, Windows, Marketplace, marketplace
+title: Configure a private marketplace for extensions
+---
+
+> **Early Access**
+>
+> Docker Extensions` private marketplace is an [early access](../../release-lifecycle#early-access-ea) feature
```
Comment (Frezzle) [inline content/desktop/extensions/private-marketplace.md:9]:
```suggestion
> Docker Extensions` private marketplace is an [early access](../../release-lifecycle#early-access-ea) feature.
```

sorry, one last nit

File: content/desktop/extensions/private-marketplace.md (hunk 2 above)
Comment (dvdksn) [reply content/desktop/extensions/private-marketplace.md:77]: the indentation looks off as well. should be two spaces

Comment (aevesdocker) [reply content/desktop/extensions/private-marketplace.md:77]: > should be two spaces
Style guide states 3 https://docs.docker.com/contribute/style/formatting/#code-and-code-samples

File: content/desktop/extensions/private-marketplace.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+- You have the ability to push the `extension-marketplace` folder and `admin-settings.json` file to the locations specified below through device management software such as [Jamf](https://www.jamf.com/).
+
+## Step one: Initialize the private marketplace
+
+1. Create a folder locally for the content that will be deployed to your developers’ machines:
+
+   ```bash
+   mkdir myMarketplace
+   cd myMarketplace
+   ```
```
[the remark this answers is not in the document]
Comment (aevesdocker) [reply content/desktop/extensions/private-marketplace.md]: > Not sure what the proper console syntax for Windows would be? 
Elsewhere in docs we've use $, so have kept it consistent

File: content/desktop/extensions/private-marketplace.md (hunk 2 above)
Comment (dvdksn) [reply content/desktop/extensions/private-marketplace.md:77]: style guide is outdated then... iirc that had something to do w/ jekyll.

File: content/desktop/extensions/private-marketplace.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+- `extensionsEnabled` enables Docker Extensions
+- `extensionsPrivateMarketplace` activates the private marketplace and ensures Docker Desktop connects to content defined and controlled by the administrator instead of the public Docker marketplace.
+- `onlyMarketplaceExtensions` allows or blocks users from installing other extensions by using the command line. Teams developing new extensions must have this setting unlocked (`"locked": false`) to install and test extensions being developed
+- `extensionsPrivateMarketplaceAdminContactURL` defines a contact link for users to request new extensions in the private marketplace. If `value` is empty then no link is shown to your developers on Docker Desktop, otherwise this can be either an HTTP link or a “mailto:” link. For example,
+
+  ```json
+  "extensionsPrivateMarketplaceAdminContactURL": {
+  "locked": true,
+  "value": "mailto:admin@acme.com"
```
Comment (dvdksn) [inline content/desktop/extensions/private-marketplace.md:80]:
```suggestion
    "locked": true,
    "value": "mailto:admin@acme.com"
```

**Review by dvdksn — APPROVED** [review]

**Pull request merged**